src/transformers/tokenization_utils.py

```diff
@@ -1292,6 +1294,15 @@ def prepare_for_model(
         len_ids = len(ids)
         len_pair_ids = len(pair_ids) if pair else 0
 
+        if return_token_type_ids is None:
+            return_token_type_ids = "token_type_ids" in self.return_outputs
+        if return_attention_mask is None:
+            return_attention_mask = "attention_mask" in self.return_outputs
+        if return_overflowing_tokens is None:
+            return_overflowing_tokens = "overflowing_tokens" in self.return_outputs
```

Comment:
Not by default, but a user could pass it as argument so that it is always returned. Not as important as the other three though, I agree. Will remove it.